src/store/modules/erc20/erc20-actions.js

```diff
@@ -14,26 +14,30 @@ const STATUS_INTERVAL = 25000
 // Setup decoder
 const abiDecoder = new AbiDecoder(Erc20)
 
-const initTransaction = (api, context, ethAddress, amount, increaseFee) => {
+const initTransaction = async (api, context, ethAddress, amount, increaseFee) => {
   const contract = new EthContract(Erc20, context.state.contractAddress)
 
+  const nonce = await api.getTransactionCount(context.state.address)
+  const gasPrice = await api.getGasPrice()
+
   const transaction = {
     from: context.state.address,
     to: context.state.contractAddress,
     value: '0x0',
     // gasLimit: api.fromDecimal(DEFAULT_ERC20_TRANSFER_GAS), // Don't take default value, instead calculate with estimateGas(transactionObject)
-    // gasPrice: context.getters.gasPrice, // Set gas price to auto calc. Deprecated after London hardfork
-    // nonce // Let sendTransaction choose it
+    gasPrice,
+    nonce,
     data: contract.methods
       .transfer(ethAddress, ethUtils.toWhole(amount, context.state.decimals))
       .encodeABI()
   }
 
-  return api.estimateGas(transaction).then((gasLimit) => {
-    gasLimit = increaseFee ? gasLimit * INCREASE_FEE_MULTIPLIER : gasLimit
-    transaction.gas = gasLimit
-    return transaction
-  })
+  const defaultGasLimit = await api.estimateGas(transaction)
```

Comment:
1. Rename `defaultGasLimit` ⟶ `gasLimit`
2. Remove `DEFAULT_ETH_TRANSFER_GAS` and `DEFAULT_ERC20_TRANSFER_GAS` from `constants/index.js`
3. `const gasLimit = api.estimateGas` or set to `CryptosInfo[${crypto}].defaultGasLimit` if unable to receive
4. Set `gasLimit` in transaction
5. Change `INCREASE_FEE_MULTIPLIER` from `2` to `1.5` with apropriate code changes

Reply:
4. Not sure what you mean. The `gasLimit` is already set.

![image](https://github.com/Adamant-im/adamant-im/assets/25831507/6861ec82-fe6d-4ccf-993d-09af0a2bdf48)

Reply:
1, 3, 5 resolved

Reply:
2- Remove, see my comment below.
4- It's done.